### PR TITLE
Bump to pg_auto_failover_enterprise 1.6.2-2

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: pkgvars [hunk 1]
```diff
@@ -2,6 +2,6 @@ rpm_pkgname=pg-auto-failover-enterprise
 deb_pkgname=auto-failover-enterprise
 hubproj=citus-ha
 pkgdesc='Postgres extension for automated failover and high-availability'
-pkglatest=1.6.2-1
+pkglatest=1.6.2-2
 releasepg=10,11,12,13
```
Comment (hanefi) [inline pkgvars]: Can we also add 14 here ?

```suggestion
releasepg=10,11,12,13,14
```

File: pg-auto-failover-enterprise.spec [hunk 2]
```diff
@@ -371,6 +371,9 @@ rm -f %{pginstdir}/bin/pg_autoctl
 
 
 %changelog
+* Fri Sep 17 2021 - Gurkan Indibay <gindibay@microsoft.com> 1.6.2-2
+- Official 1.6.2 release of Pg Auto Failover Enterprise
```
Comment (hanefi) [inline pg-auto-failover-enterprise.spec]: We can have a better description here for future reference.

```suggestion
- Removes encryption from packages
```

Comment (gurkanindibay) [reply pg-auto-failover-enterprise.spec]: Fixed

File: pkgvars (hunk 1 above)
Comment (gurkanindibay) [reply pkgvars]: Fixed

**Review by hanefi — APPROVED** [review]

**Pull request merged**